### PR TITLE
Helper branch for EZP-24869

### PR DESCRIPTION
**Pull request by bdunogier**
See https://github.com/ezsystems/ezpublish-kernel/pull/1425

### CODE REVIEW

File: doc/apache2/vhost.template [hunk 1]
```diff
@@ -52,7 +52,7 @@
     # Whether to use debugging.
     # Possible values: 0 or 1
     # Defaults to 0 if omitted, unless ENVIRONMENT is set to: "dev"; supported on 5.2 and higher
-    #SetEnv USE_DEBUGGING 0
+    SetEnv USE_DEBUGGING 1
```
Comment (andrerom) [inline doc/apache2/vhost.template]: should be omitted from merge

Comment (bdunogier) [inline doc/apache2/vhost.template]: Well, no...

Without debugging, we won't get any feedback from errors. It is required by the exceptions parsing added to ezsystems/BehatBundle#34

Why should we absolutely disable it ?

Comment (bdunogier) [inline doc/apache2/vhost.template]: Added a `--use-debugging` option to the `generatevhost.php` script & `vhost.template`. Defaults to 0, set to 1 by `prepare_apache2.sh`.

Does that work for you @andrerom ?

Comment (andrerom) [inline doc/apache2/vhost.template]: better like that
But personally I think prod is what we should test with behat, faster and closer to how it is meant to be run in prod, but I also see the aspect of exceptions detection, so this is ok for now so we can easier spot regressions.

**Comment by andrerom** [comment]
+1

File: doc/apache2/vhost.template (hunk 1 above)
Comment (bdunogier) [inline doc/apache2/vhost.template]: > But personally I think prod is what we should test with behat, faster and closer to how it is meant to be run in prod

We can do prod + debugging, no problem, they're different settings. Debugging enables stacktraces and a couple other features, but doesn't enforce dev mode at all.

Right now, the `behat` env we use is kind of dev, but mostly with prod settings: https://github.com/ezsystems/ezplatform/blob/master/ezpublish/config/config_behat.yml (inherits config_dev, but forces most settings to their prod value).

Comment (andrerom) [inline doc/apache2/vhost.template]: yes, improvement potentials. Started with the test stuff as basis from symfony-standard, but applies badly to a functional testing use case.

**Pull request merged**